web_server/authors/views.py

```diff
@@ -8,45 +8,41 @@
 from django.utils.timezone import make_aware
 from django.urls import reverse
 
+
 import json
 import datetime
 import sys
-
+import re
 from social_distribution.utils.endpoint_utils import Endpoint, Handler
-
-
+# used for stripping url protocol
+url_regex = re.compile(r"(http(s?))?://")
 # Ida Hou
 # return a list of author id that are currently stored in database and
 # are not friend with current author
+
+
 def view_list_of_available_authors_to_befriend(request, author_id):
     if request.method != 'GET':
         return HttpResponse("Method not Allowed", status=405)
```

Comment:
Did you see the Endpoint and Handler classes? They abstract the simple error handling like this away so your handlers become more compact. Is there a reason you didn't or don't want to use them?